BUILD.bazel

```diff
@@ -0,0 +1,407 @@
+        ":linux_x86_64": [],
+    }),
+    copts = all_options,
+    defines = DEFINES,
+    includes = [
+        ilm_thread_path,
+    ],
+    linkopts = select({
+        ":windows_x86_64": [],
+        ":linux_x86_64": ["-pthread"],
+    }),
+    deps = [
+        ":iex",
+        ":ilmbase_config",
+    ],
+)
+
+################################################################################
+# Build the Imath library
+imath_path = "IlmBase/Imath/"
+
+cc_library(
+    name = "imath",
+    srcs = glob([imath_path + "*.cpp"]),
+    hdrs = glob([imath_path + "*.h"]),
+    copts = all_options,
+    defines = DEFINES,
+    includes = [
+        imath_path,
+    ],
+    deps = [
+        ":half",
+        ":iex",
+        ":iex_math",
+        ":ilmbase_config",
+    ],
+)
+
+################################################################################
+# Build the IlmImf library
+ilm_imf_path = "OpenEXR/IlmImf/"
+
+ilm_imf_util_path = "OpenEXR/IlmImfUtil/"
+
+# Generates the config headers. Note we don't use the autoconf, but the
+# libraries still require a header.
+genrule(
+    name = "configure_openexr",
+    outs = ["OpenEXR/config/OpenEXRConfig.h"],
+    cmd = "touch $@",
+)
+
+genrule(
+    name = "configure_openexr2",
+    outs = ["OpenEXR/config/OpenEXRConfigInternal.h"],
+    cmd = "touch $@",
+)
+
+# Wraps the config auto generation with the include folder configuration.
+cc_library(
+    name = "openexr_config",
+    hdrs = [
+        ":configure_openexr",
+        ":configure_openexr2",
+    ],
+    defines = DEFINES,
+    includes = ["OpenEXR/config/"],
+)
+
+cc_binary(
+    name = "dwaLookups",
+    srcs = [
+        ilm_imf_path + "dwaLookups.cpp",
+    ] +
+    # Note we include various headers via glob since we don't have cc_library
+    # targets for some of the sub-libraries of OpenEXR.
+    glob(
+        [
+            ilm_imf_util_path + "**/*.h",
+            ilm_imf_path + "*.h",
+        ],
+        exclude = [
+            ilm_imf_path + "dwaLookups.h",
+            ilm_imf_path + "b44ExpLogTable.h",
+        ],
+    ),
+    copts = all_options,
+    defines = DEFINES,
+    includes = [
+        imath_path,
+        iex_math_path,
+        ilm_imf_path,
+        ilm_imf_util_path,
+    ],
+    deps = [
+        ":half",
+        ":iex",
+        ":ilm_thread",
+        ":imath",
+        ":openexr_config",
+    ],
+)
+
+cc_binary(
+    name = "b44ExpLogTable",
+    srcs = [
+        ilm_imf_path + "b44ExpLogTable.cpp",
+    ] + glob(
+        ["**/*.h"],
+        exclude = [ilm_imf_path + "b44ExpLogTable.h"],
+    ),
+    copts = all_options,
+    defines = DEFINES,
+    includes = [
+        half_path,
+        imath_path,
+        iex_math_path,
+        iex_path,
+        ilm_imf_path,
+        ilm_imf_util_path,
+        ilm_thread_path,
+    ],
+    deps = [
+        ":half",
+        ":iex",
+        ":ilm_thread",
+        ":openexr_config",
+    ],
+)
+
+genrule(
+    name = "dwaLookupsAutoGen",
```

Comment:
last year, we checked in the secondary generated files, so this generation step should not be necessary. Omitting is probably better to reduce the maintenance footprint.

Reply:
Fixed